kornia/color/rgb.py

```diff
@@ -2,13 +2,57 @@
 import torch.nn as nn
 
 
+class RgbToRgba(nn.Module):
+    r"""Convert image from RGB to RGBA.
+
+    Add an Alpha channel to existing RGB image.
+
+    args:
+        image (torch.Tensor): RGB image to be converted to RGBA.
+        alpha_val (int): An integer between 0 to 255.
+
+    returns:
+        torch.Tensor: RGBA version of the image.
+
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 4, H, W)`
+
+    Examples::
+
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> rgba = kornia.color.RgbToRgba(alpha_val)
+        >>> output = rgba(input)  # 2x4x4x5
+    """
+
+    def __init__(self) -> None:
+        super(RgbToRgba, self).__init__()
+
+    def forward(self, image: torch.Tensor, alpha_val: float) -> torch.Tensor:  # type: ignore
+        return rgb_to_rgba(image, alpha_val)
```

Comment:
```python
    def __init__(self, alpha_val: float) -> None:
        super(RgbToRgba, self).__init__()
        self.alpha_val: float = alpha_val

    def forward(self, image: torch.Tensor) -> torch.Tensor:  # type: ignore
        return rgb_to_rgba(image, self.alpha_val)
```

Reply:
ah yes , forgot putting the alpha_val at constructor